### ISSUE TITLE
Permettre connexion OIDC directe 

### ISSUE THREAD

**Issue by bzg** [codegouvfr/sill-web#98]

**Comment by garronej** [comment]
With https://docs.oidc-spa.dev/getting-started/overview

**Comment by garronej** [comment]
Le nom d'organisation est actuellement stoqué et mis a jour via Keyclaok, il faudrais stoquer dans notre base pour avoir un système générique.

**Comment by JeromeBu** [comment]
On a constaté avec @garronej que l'API de keycloak est utilisé pour récupéré des données statistiques, et donc on ne peux pas éléminer aussi facilement la connexion par keycloak. (Je ne vais pas creuser plus pour l'instant, et regarder plutôt la partie intégration d'une autre source de donnée : HAL)

**Comment by bzg** [comment]
Je ne connais pas le statut pour cette issue mais je décale à 1.6 - @JeromeBu je te laisse fermer si pertinent.

**Comment by bzg** [comment]
Pour mémoire, suite à l'échange de ce matin, il y a ce besoin de ne pas stocker les informations d'affiliation dans keycloak.

**Comment by JeromeBu** [comment]
Yes je crée une issue à ce propos, il faut qu'on élimine toutes les dépendences direct à keycloak.

Permier sujet à traiter :
- [ ] Stocker l'organisme des utilisateurs dans le SILL et plus dans keycloak
